lib/lpc55-flash/src/lib.rs

```diff
@@ -156,3 +156,15 @@ pub unsafe extern "C" fn __write_block(
 
     HypoStatus::Success
 }
+
+#[no_mangle]
+pub unsafe extern "C" fn __active_slot() -> UpdateTarget {
+    let current = this_image!();
+    if image_a_base!() == current {
+        UpdateTarget::ImageA
+    } else if image_b_base!() == current {
+        UpdateTarget::ImageB
+    } else {
```

Comment:
Test against image_stage0_base

Reply:
Fixed in https://github.com/oxidecomputer/hubris/pull/991/commits/34fa9b3c7bfefaf747c7161db49937c79615f8c6